soc/arm/st_stm32/stm32g0/Kconfig.defconfig.series

```
@@ -13,9 +13,6 @@ config SOC_SERIES
 	default "stm32g0"
 
 if PM
-config PM_DEVICE
-	default y
-
 config STM32_LPTIM_TIMER
 	default y
 endif # PM
```

I'd suggest to use the 2 lines version (to be applied everywhere)
```suggestion
config STM32_LPTIM_TIMER
	default y if PM
```

---

adjusted